### DISCUSSION TITLE
[UI] "Hide" Product in App

### DISCUSSION THREAD

**Discussion by ghost** [btcpayserver/btcpayserver#2827] [Ideas / Feature Requests]
In the Settings of an App, I can Generate new products, and delete them. There is however no way, to "temporarily hide" a product. 

It would be useful, if there were a button with "Hide" / "Show" a certain product. This way it can be easily paused from public offering, and activated again later when ready.

**Comment by bolatovumar** [comment]
Seems like a reasonable use case.

**Reply by bolatovumar** [reply]
@MaxHillebrand PR up here: https://github.com/btcpayserver/btcpayserver/pull/2842

**Reply by ghost** [reply]
3 hours from cACK to writing the code.

Damn - this guy ships!

Thanks @bolatovumar :black_heart:
